frontend/src/services/grandmaster-manager/index.ts

```diff
@@ -12,26 +12,27 @@ import { StatsServiceClient } from './statsServiceClient';
 export interface AccountDetails {
   accountAddress: string;
   balance: number;
-  networkId: number;
-  rpcAddress: string;
-  denom: string;
 }
 
 interface GrandMasterInfo {
-  grandMasterAddress: string;
-  networkId: number;
+  grandMasterAddress: string[];
+  minimumDelegation: number;
+}
+
+export interface ChainSettingInfo {
   rpcUrl: string;
   denom: string;
-  minimumDelegation: number;
+  chainId: number;
 }
 
 export class GrandMasterManager {
-  private grandMaster: GrandMasterInfo | undefined;
+  private grandMasterInfo: GrandMasterInfo | undefined;
   private web3Client: Web3 | undefined;
   private web3Contract: any;
   private statsServiceClient: StatsServiceClient;
+  private chainSetting: ChainSettingInfo;
 
-  constructor() {
+  constructor(chainSetting?: ChainSettingInfo) {
```

Comment:
@trust2065 could u pass in those values when create `GrandMasterManager`? u can get it via the new chainsetting API after we deployed it

Reply:
the `ManagementLoginPage.tsx` will be broken for the time being